packages/fx-core/src/plugins/resource/function/plugin.ts

```diff
@@ -636,6 +641,64 @@ export class FunctionPluginImpl {
     return ResultFactory.Success();
   }
 
+  public async generateArmTemplates(ctx: PluginContext): Promise<FxResult> {
+    const selectedPlugins = (ctx.projectSettings?.solutionSettings as AzureSolutionSettings)
+      .activeResourcePlugins;
+    const context = {
+      plugins: selectedPlugins,
+    };
+
+    const bicepTemplateDirectory = path.join(
+      getTemplatesFolder(),
+      "plugins",
+      "resource",
+      "function",
+      "bicep"
+    );
+
+    const moduleTemplateFilePath = path.join(bicepTemplateDirectory, Bicep.moduleTemplateFileName);
+
+    const parameterTemplateFilePath = path.join(
+      bicepTemplateDirectory,
+      Bicep.inputParameterOrchestrationFileName
+    );
+    const resourceTemplateFilePath = path.join(
```

Comment:
Please rename the variable as `moduleOrchestrationFilePath` to make it consistent. (It is `module` instead of `resource`)